packages/connectivity/src/scp-cf/destination/destination-cache.spec.ts

```diff
@@ -67,6 +69,12 @@ const destinationOne: Destination = {
   isTrustingAllCertificates: false
 };
 
+// Cache with expiration time
+const testCacheOne = new TestCache(30);
+
+// Setting the destinationCache with custom class instance
+setDestinationCache(testCacheOne);
```

Comment:
[req] I think this will become relevant before merging, but I leave the comment anyway. Think there should be additional test regarding the custom inplemenation so keep the old test but in a separate `describe` do some tests where you set the custom implementation and assert that this implementation is used for all used cases like set,get and clear. This should be possible via spy I guess.